<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab</class>
 <widget class="QMainWindow" name="Tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    border-top: 2px solid rgba(0, 0, 0, 1); /* 黑色 */
}

QTabBar::tab {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 12px;
    font-weight: 500;
    padding: 10px 25px; /* 增加内边距 */
    border-bottom: none;
	min-width: 45px;
   
}

QTabBar::tab:selected {
    color: rgba(123,190,254, 1); /* 淡蓝色 */
    border-bottom: 2px solid rgba(123,190,254, 1); /* 淡蓝色 */
}

QTabBar::tab:!selected {
    margin-bottom: 2px;
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="t1_label_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>这里是tab1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="t1_switch_t2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>Switch To Tab 2</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="t2_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>这里是tab2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="t2_swicth_t1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>281</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;</string>
      </property>
      <property name="text">
       <string>Switch To Tab 1</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
